functional_tests/test_db_online_offline.py

```diff
@@ -18,7 +19,7 @@
 import logging
 log = logging.getLogger(lib.settings.LOGGER)
 
-NUM_ENDPOINTS = 10
+NUM_ENDPOINTS = 13
 
 def rest_scan(sync_gateway, db, online, num_docs, user_name, channels):
```

Comment:
I think applying this function again should yield the same result as the name applies. However the implementation doesn't support the above behavior.